Validation/Configuration/python/postValidation_cff.py

```diff
@@ -116,9 +117,14 @@
 _phase2_postValidation += MuonME0DigisPostProcessors
 _phase2_postValidation += MuonME0SegPostProcessors
 
+_phase1_postValidation = postValidation.copy()
+_phase1_postValidation += siPixelPhase1OfflineDQM_harvestingV
+
 from Configuration.Eras.Modifier_run2_GEM_2017_MCTest_cff import run2_GEM_2017_MCTest
 run2_GEM_2017_MCTest.toReplaceWith( postValidation, _run3_postValidation )
 from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
 run3_GEM.toReplaceWith( postValidation, _run3_postValidation )
 from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
 phase2_hgcal.toReplaceWith( postValidation, _phase2_postValidation )
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+phase1Pixel.toReplaceWith( postValidation, _phase1_postValidation )
```

Comment:
Before I commit the changes, was this the sort of change you had in mind?

i.e. we check if it's phase 1 pixel first, but then replace that validation sequence with the Run 3 one if it is Run 3?

```
_phase1_postValidation = postValidation.copy()
_phase1_postValidation += siPixelPhase1OfflineDQM_harvestingV

_run3_postValidation = postValidation.copy()
_run3_postValidation += MuonGEMHitsPostProcessors
_run3_postValidation += MuonGEMDigisPostProcessors
_run3_postValidation += MuonGEMRecHitsPostProcessors
_run3_postValidation += siPixelPhase1OfflineDQM_harvestingV

_phase2_postValidation = _run3_postValidation.copy()
_phase2_postValidation += hgcalPostProcessor
_phase2_postValidation += MuonME0DigisPostProcessors
_phase2_postValidation += MuonME0SegPostProcessors

from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
phase1Pixel.toReplaceWith( postValidation, _phase1_postValidation )
from Configuration.Eras.Modifier_run2_GEM_2017_MCTest_cff import run2_GEM_2017_MCTest
run2_GEM_2017_MCTest.toReplaceWith( postValidation, _run3_postValidation )
from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
run3_GEM.toReplaceWith( postValidation, _run3_postValidation )
from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
phase2_hgcal.toReplaceWith( postValidation, _phase2_postValidation )
```